lerobot/common/datasets/lerobot_dataset.py

```diff
@@ -983,7 +983,7 @@ def __init__(
         root: str | Path | None = None,
         episodes: dict | None = None,
         image_transforms: Callable | None = None,
-        delta_timestamps: dict[list[float]] | None = None,
+        delta_timestamps: dict[str, dict[list[float]]] | None = None,
```

Comment:
not sure if this is the way you'd implement delta timestamps? dict of repo ID to the timestamps?